js-data/js-data.d.ts

```diff
@@ -0,0 +1,404 @@
+// Type definitions for JSData v1.3.0
+// Project: https://github.com/js-data/js-data
+// Definitions by: Stefan Steinhart <https://github.com/reppners>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+///////////////////////////////////////////////////////////////////////////////
+// Promises in js-data are ES6 polyfill promises
+///////////////////////////////////////////////////////////////////////////////
+
+/// <reference path="../es6-promise/es6-promise.d.ts" />
+
+///////////////////////////////////////////////////////////////////////////////
+// js-data module (js-data.js)
+///////////////////////////////////////////////////////////////////////////////
+
+// defining what exists in JSData and how it looks
+declare module JSData_ {
```

Comment:
There is a name collision because js-data defines itself in the global browser namespace as property 'JSData'. [see here](https://github.com/reppners/DefinitelyTyped/commit/143ee58d484a6fca6f59314ed141a16d55535fc7#diff-869a01d725fdd226b7f8f892445661ddR396)

I tried to make the tests pass with [this version](https://github.com/reppners/DefinitelyTyped/commit/4fb28c2c4aef8c4a3b6558cc11b6961b37fb13fd) but couldn't get it to work.

Do you have a suggestion to solve this problem in a better way than renaming the module?